core/embed/extmod/modtrezorio/modtrezorio-poll.h

```diff
@@ -52,7 +52,7 @@ STATIC mp_obj_t mod_trezorio_poll(mp_obj_t ifaces, mp_obj_t list_ref,
     mp_raise_TypeError("invalid list_ref");
   }
 
-  const mp_uint_t timeout = trezor_obj_get_uint(timeout_ms);
+  const mp_uint_t timeout = trezor_obj_get_int(timeout_ms);
```

Comment:
If we are going to proceed with this fix I think this really deserves a comment in the source code describing why this is intended.